src/main/java/com/exasol/validation/ValidationResult.java

```diff
@@ -0,0 +1,55 @@
+package com.exasol.validation;
+
+/**
+ * Represents a result of a single validation.
+ */
+public class ValidationResult {
```

Comment:
I'am not sure if an interface with two implementing classes would fit better here.

Reply:
I was thinking about it, but for now one class is enough and  I can easily replace it later, as it's hidden deep in the implementation